devices/Sen5x/Sen5xSensor.cs

```diff
@@ -0,0 +1,355 @@
+        public void StartMeasurement()
+        {
+            Read(0x0021, TimeSpan.FromMilliseconds(50));
+        }
+
+        /// <summary>
+        /// Starts a continuous measurement without PM. Only humidity, temperature, VOC, and NOx are available in this mode. Laser and fan are switched off to keep power consumption low.
+        /// </summary>
+        public void StartMeasurementWithoutPM()
+        {
+            Read(0x0037, TimeSpan.FromMilliseconds(50));
+        }
+
+        /// <summary>
+        /// Stops the measurement. Use this command to return to the initial state (Idle-Mode).
+        /// </summary>
+        public void StopMeasurement()
+        {
+            Read(0x0104, TimeSpan.FromMilliseconds(200));
+        }
+
+        /// <summary>
+        /// This command can be used for polling to find out when new measurements are available.
+        /// </summary>
+        /// <returns>True when data is ready to be read, false otherwise.</returns>
+        public bool ReadDataReadyFlag()
+        {
+            var data = Read(0x0202, TimeSpan.FromMilliseconds(20), new byte[3]);
+            return data[1] > 0;
+        }
+
+        /// <summary>
+        /// Reads the measured values from the sensor module and resets the "Data-Ready Flag". If the sensor module is in Measurement-Mode, an updated measurement value is provided
+        /// every second and the "Data-Ready Flag" is set. If no synchronized readout is desired, the "Data-Ready Flag" can be ignored. The command "Read Measured Values" always returns
+        /// the latest measured values. In RHT/Gas-Only Measurement Mode, the PM output is 0xFFFF. If any value is unknown, 0xFFFF is returned.
+        /// </summary>
+        /// <returns>The measured values.</returns>
+        public Measurement ReadMeasurement()
+        {
+            return (Measurement)Read(0x03c4, TimeSpan.FromMilliseconds(20), new Measurement());
+        }
+
+        /// <summary>
+        /// These commands allow to compensate temperature effects of the design-in at customer side by applying a custom temperature offset to the ambient temperature.
+        /// </summary>
+        /// <returns>The current temperature compensation parameters.</returns>
+        public TemperatureCompensationParameters ReadTemperatureCompensationParameters()
+        {
+            return (TemperatureCompensationParameters)Read(0x60b2, TimeSpan.FromMilliseconds(20), new TemperatureCompensationParameters());
+        }
+
+        /// <summary>
+        /// These commands allow to compensate temperature effects of the design-in at customer side by applying a custom temperature offset to the ambient temperature.
+        /// </summary>
+        /// <param name="value">The new temperature compensation parameters to be set.</param>
+        public void WriteTemperatureCompensationParameters(TemperatureCompensationParameters value)
+        {
+            Write(0x60b2, TimeSpan.FromMilliseconds(20), value);
```

Comment:
And of course, same remark for the write function and for the read one. Use an internal enum, I guess can be the same one.